tests/integration_tests.rs

```diff
@@ -93,6 +93,144 @@ fn line_range_last_3() {
         .stdout("line 2\nline 3\nline 4\n");
 }
 
+#[test]
+fn tabs_passthrough_wrapped() {
+    bat()
+        .arg("tabs.txt")
+        .arg("--tabs=0")
+        .arg("--wrap=character")
+        .arg("--style=plain")
```

Comment:
I think the current tests also test `--tabs` in combination with other styles, right? I think we should at least test one style like `--style=numbers` here, because it will affect the tab-stops.

Reply:
I realized a trailing space would be needed to make [Test 'tabs' with 'numbers'](https://github.com/sharkdp/bat/pull/356/commits/9a42fcb74d1604dddd43466ecd52b6118192d61c) pass. So I just [removed the empty line in snapshot](https://github.com/sharkdp/bat/pull/356/commits/b7620066e107c44849a0d7c8e8544dc69e3e35b6)